lib/Sema/CSSimplify.cpp

```diff
@@ -4986,6 +4987,15 @@ ConstraintSystem::simplifyRestrictedConstraint(
   llvm_unreachable("Unhandled SolutionKind in switch.");
 }
 
+static bool isAugmentingFix(ConstraintFix *fix) {
+  switch (fix->getKind()) {
+    case swift::constraints::FixKind::TreatRValueAsLValue:
```

Comment:
Does it really need to be qualified completely like that here?

Reply:
Nope. I didn't notice the overly helpful autocomplete.